main.py

```diff
@@ -85,6 +86,8 @@ def predicate(interaction: Interaction):
 @bot.event
 async def on_ready():
     print(f"{bot.user} has connected to Discord!")
+    bot.server_embed = await tb_embeds.ServersEmbed.create(bot.get_channel(1099805791487266976))
+    await bot.tree.sync()
```

Comment:
Same as the above comment - syncing the tree automatically is unnecessary anyway.